google-cloud-spanner/src/main/java/com/google/cloud/spanner/spi/v1/GapicSpannerRpc.java

```diff
@@ -143,69 +142,69 @@ public GapicSpannerRpc(SpannerOptions options) throws IOException {
-                .setManagedChannel(options.getRpcChannels().get(0))
-                .build());
+        InstantiatingGrpcChannelProvider
+            .newBuilder()
+            .setEndpoint(options.getEndpoint())
+            .setMaxInboundMessageSize(MAX_MESSAGE_SIZE)
+            .setPoolSize(options.getNumChannels())
+            .setInterceptorProvider(new SpannerInterceptorProvider())
+            .build();
+
     CredentialsProvider credentialsProvider =
         GrpcTransportOptions.setUpCredentialsProvider(options);
-    
+
     // Disabling retry for now because spanner handles retry in SpannerImpl.
     // We will finally want to improve gax but for smooth transitioning we
     // preserve the retry in SpannerImpl
     try {
       // TODO: bump the version of gax and remove this try-catch block
       // applyToAllUnaryMethods does not throw exception in the latest version
       this.stub =
-        GrpcSpannerStub.create(
-            SpannerStubSettings.newBuilder()
-                .setTransportChannelProvider(channelProvider)
-                .setCredentialsProvider(credentialsProvider)
-                .applyToAllUnaryMethods(
-                    new ApiFunction<UnaryCallSettings.Builder<?, ?>, Void>() {
-                      @Override
-                      public Void apply(UnaryCallSettings.Builder<?,?> builder) {
-                        builder.setRetryableCodes(ImmutableSet.<StatusCode.Code>of());
-                        return null;
-                      }
-                    })
-                .build());
-    
-    this.instanceStub =
-        GrpcInstanceAdminStub.create(
-            InstanceAdminStubSettings.newBuilder()
-                .setTransportChannelProvider(channelProvider)
-                .setCredentialsProvider(credentialsProvider)
-                .applyToAllUnaryMethods(
-                    new ApiFunction<UnaryCallSettings.Builder<?, ?>, Void>() {
-                      @Override
-                      public Void apply(UnaryCallSettings.Builder<?,?> builder) {
-                        builder.setRetryableCodes(ImmutableSet.<StatusCode.Code>of());
-                        return null;
-                      }
-                    })
-                .build());
-    this.databaseStub =
-        GrpcDatabaseAdminStub.create(
-            DatabaseAdminStubSettings.newBuilder()
-                .setTransportChannelProvider(channelProvider)
-                .setCredentialsProvider(credentialsProvider)
-                .applyToAllUnaryMethods(
-                    new ApiFunction<UnaryCallSettings.Builder<?, ?>, Void>() {
-                      @Override
-                      public Void apply(UnaryCallSettings.Builder<?,?> builder) {
-                        builder.setRetryableCodes(ImmutableSet.<StatusCode.Code>of());
-                        return null;
-                      }
-                    })
-                .build());
+          GrpcSpannerStub.create(
```

Comment:
Why this whitespace change, did it not use 2-space before?

Reply:
It was wrong before :( (Note that `this.instanceStub` used to have the same indentation as `try {`

Reply:
In that case, LGTM